### PR TITLE
CS fixes

### PR DESCRIPTION
**Pull request by fabpot**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

File: src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php [hunk 1]
```diff
@@ -109,15 +109,15 @@ public static function register($mode = false)
                     return "\x1B[{$color}m{$str}\x1B[0m";
                 };
             } else {
-                $colorize = function ($str) {return $str;};
+                $colorize = function ($str) { return $str; };
```
Comment (fabpot) [inline src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php]: @nicolas-grekas This one is for you :) You made a bunch of them.

**Comment by chalasr** [comment]
While you're there, I spotted a typo in https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/DependencyInjection/Definition.php#L161, it should be ` Gets the service that this service is decorating`. Maybe could you fix it btw?

**Comment by fabpot** [comment]
@chalasr done, thanks

File: src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/default.phpt [hunk 2]
```diff
@@ -31,14 +31,14 @@ class FooTestCase
     public function testLegacyFoo()
     {
         @trigger_error('silenced foo deprecation', E_USER_DEPRECATED);
-        trigger_error('unsilenced foo deprecation', E_USER_DEPRECATED);
-        trigger_error('unsilenced foo deprecation', E_USER_DEPRECATED);
+        @trigger_error('unsilenced foo deprecation', E_USER_DEPRECATED);
+        @trigger_error('unsilenced foo deprecation', E_USER_DEPRECATED);
```
Comment (jakzal) [inline src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/default.phpt]: Wasn't this suppose to be an unsilenced deprecation?

Comment (fabpot) [inline src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/default.phpt]: I messed up with my second commit, but I've reverted that afterwards. Sorry for the noise.

**Pull request merged**